frontend/src/app/app.module.ts

```diff
@@ -1,14 +1,18 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-import { FormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { HttpClientModule } from '@angular/common/http';
+import { MAT_DIALOG_DEFAULT_OPTIONS, MatDialogModule } from '@angular/material/dialog';
```

Comment:
Missed this the first time, but should shift these to a module for material imports and provide in root as in the PR for collab, perhaps in this PR if collab is merged before this one, or in that one if this is merged first

Reply:
Sure, if that's the case then we should probably just merge collab first.